packages/recomponents/src/components/mixins/capitalize-the-first-letter-in-slot-mixin.js

```diff
@@ -0,0 +1,20 @@
+export default {
+    methods: {
+        capitalizeTheFirstLetterInSlot() {
+            if (this.$slots.default) {
+                let {text} = this.$slots.default[0];
+                if (text) {
+                    text = text.trim();
+                    text = text.charAt(0).toUpperCase() + text.slice(1);
+                    this.$slots.default[0].text = text;
+                }
+            }
+        },
+    },
```

Comment:
This only work if slot's content is plain text

```html
<r-icon-button>
    icon button
</r-icon-button>
```

But if I'd like to wrap it with some tag it doesn't work:

```html
<r-icon-button>
    <strong>icon button</strong> 
</r-icon-button>
```

Reply:
Any way I think it's better to proceed with CSS only instead of handing all possible inputs in JS parser

Reply:
+1

Reply:
New PR: https://github.com/Rebilly/rebilly-recomponents/pull/185, thanks for helping me a lot!